runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -559,18 +559,127 @@ xlpSubOptionsParser(J9JavaVM *vm, IDATA xlpIndex, XlpError *xlpError, UDATA *req
 	return xlpErrorState;
 }
 
+/**
+ * 
+ * Given the requested size of the large pages to be used, the method will verify that value is supported by the system. 
+ * If the value is not supported, the system will set the largest available size instead. The user will also be notified 
```

Comment:
This comment is misleading. There would be not a `largest available size` but default Large Page size if any

Reply:
Thanks for catching that. 

See https://github.com/eclipse/openj9/pull/7476/commits/8010113e300fbff7aafb6872651928d4966df751